Modules/Relationships/src/Relationships.ConsumerApi/Controllers/RelationshipsController.cs

```diff
@@ -130,6 +131,16 @@ public async Task<IActionResult> RelationshipReactivationRequest([FromRoute] str
         var response = await _mediator.Send(new RequestRelationshipReactivationCommand { RelationshipId = id }, cancellationToken);
         return Ok(response);
     }
+
+    [HttpPut("{id}/Decompose")]
+    [ProducesResponseType(typeof(HttpResponseEnvelopeResult<DecomposeRelationshipResponse>), StatusCodes.Status200OK)]
+    [ProducesError(StatusCodes.Status400BadRequest)]
+    [ProducesError(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> DecomposeRelationship([FromRoute] string id, CancellationToken cancellationToken)
+    {
+        await _mediator.Send(new DecomposeRelationshipCommand() { RelationshipId = id }, cancellationToken);
+        return NoContent();
```

Comment:
The `ProducesResponseTypeAttribute` states that this route returns a `HttpResponseEnvelopeResult<DecomposeRelationshipResponse>` and a status code 200 (which is correct), but you actually return `NoContent`.

Reply:
Sorry about this oversight. Changes made.